spec/person_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative '../person'
+
+describe Person do
+  person = Person.new(23)
+  it 'Shoud create a person' do
+    expect(person.age).to eql 23
+  end
+
+  it 'Shoud check if person has no name' do
+    expect(person.name).to eql 'Unknown'
+  end
+
+  it 'Shoud check if person has no name' do
+    expect(person.name).not_to eql 'Something'
+  end
+
+  it 'Shoud Correct the name' do
+    person = Person.new(30, 'mhdez')
+    person.validate_name
+    expect(person.name).to eql 'Mhdez'
+  end
+
+  it 'Shoud check if person has parent permission' do
+    expect(person.parent_permission).to eq 'true'
+  end
```

Comment:
- [ ] try to be as thorough as possible when testing, for example here make sure to test positive and negative cases

Reply:
@mikethreels , thanks for the review but I still do not understand what you mean by positive and negative case, Could elaborate, if possible with an example

Reply:
What I meant is, you wrote a test to check if everything works fine when the parent permission is true but you should write a test to check what happens when the parent permission is false as well.